x-pack/test/functional/apps/aiops/log_pattern_analysis_in_discover.ts

```diff
@@ -23,6 +22,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     });
   }
 
+  // temporarily disabled, to be replaced
```

Comment:
Is this comment still applicable?

Reply:
Removed in https://github.com/elastic/kibana/pull/178916/commits/d5ed385c8fafaec9a24149e96491d154787e01a7